src/test/debuginfo/union-smoke.rs

```diff
@@ -9,7 +9,6 @@
 // except according to those terms.
 
 // min-lldb-version: 310
-// ignore-macos FIXME(#37479)
```

Comment:
Note that while I think this should be fixed for macos, I've only tested for linux with llvm 3.9.